tools/Replay.hpp

```diff
@@ -163,6 +271,17 @@ struct Stats {
     {
         const auto now = std::chrono::steady_clock::now();
         const size_t runMs = std::chrono::duration_cast<std::chrono::milliseconds>(now - _start).count();
+
+        CPUStat finalCPUStat = GetProcessTimes();
+        double userCPUTimeDiff = finalCPUStat.userCPUTime - initialCPUStat.userCPUTime;
+        double kernelCPUTimeDiff = finalCPUStat.kernelCPUTime - initialCPUStat.kernelCPUTime;
+        double elapsedTimeDiff = finalCPUStat.elapsedTime - initialCPUStat.elapsedTime;
+        double cpuUsagePercentage = ((userCPUTimeDiff + kernelCPUTimeDiff) * 100) / elapsedTimeDiff;
+
+        std::cout << "Peak memory usage: " << _peakMemoryUsage << "kB";
+
+        std::cout << "Percentage of CPU: " << cpuUsagePercentage << "%\n";
```

Comment:
A %age like this is not useful; please drop. We want raw CPU cycle times spent as per the timer wrapper I included.